src/glewlwyd.h

```diff
@@ -32,17 +32,28 @@
 #include <jansson.h>
 #include <jwt.h>
 
-#define _GNU_SOURCE
-#define __USE_GNU
+#ifndef _GNU_SOURCE
+ #define _GNU_SOURCE
+#endif
+
+#ifndef __USE_GNU
+ #define __USE_GNU
+#endif
+
```

Comment:
I like this change but I think it doesn't fit in the pull request, I suggest you move this change in a new pull request if you don't mind